artifacts/alfresco_setup.properties

```diff
@@ -0,0 +1,8 @@
+http://ec2-54-224-31-118.compute-1.amazonaws.com:8080,admin,admin
+yasser
```

Comment:
The values in this file seem to very "Yasser-specific"... Can you put some more generic values?